framework/py/flwr/supernode/start_client_internal.py

```diff
@@ -86,6 +86,7 @@ def start_client_internal(
     isolation: str = ISOLATION_MODE_SUBPROCESS,
     clientappio_api_address: str = CLIENTAPPIO_API_DEFAULT_SERVER_ADDRESS,
     health_server_address: Optional[str] = None,
+    trust_entity: Optional[list] = None,
```

Comment:
Would this be a `list`? or better `dict[str, bytes]`, where `str` is the entity_id and `bytes` is the public key?

Also, should we use `trust_entities`?